cli/cobra.go

```diff
@@ -159,15 +161,27 @@ func isFirstLevelCommand(cmd *cobra.Command) bool {
 	return cmd.Parent() == cmd.Root()
 }
 
-func commandVendor(cmd *cobra.Command) string {
-	width := 13
-	if v, ok := cmd.Annotations[pluginmanager.CommandAnnotationPluginVendor]; ok {
-		if len(v) > width-2 {
-			v = v[:width-3] + "…"
+func isPluginFirstLevelCommand(cmd *cobra.Command) bool {
+	return isPlugin(cmd) && isFirstLevelCommand(cmd)
+}
+
+func decoratedCommandName(cmd *cobra.Command) string {
+	decoration := " "
+	if isPluginFirstLevelCommand(cmd) {
+		decoration = "*"
+	}
+	return cmd.Name() + decoration + strings.Repeat(" ", cmd.NamePadding()-len(cmd.Name()))
+}
+
+func commandVendorAndVersion(cmd *cobra.Command) string {
```

Comment:
Same as my other comment; I think it's implicit already that it's a command, so could be just

```suggestion
func vendorAndVersion(cmd *cobra.Command) string {
```

Reply:
Done